site/content/guides/routing.md

```diff
@@ -0,0 +1,238 @@
+
+const RouterApp = createApp({
+  name: 'RouterApp',
+  providers: [
+    {
+      name: 'component',
+      useValue: RootComponent
+    },
+    {
+      name: 'router',
+      useValue: router,
+      cascade: true,
+    },
+  ],
+});
+```
+
+### Wiring things up
+
+We have everything ready to start the app. We will need `frint-react` to render our app into HTML
+element (with id `root` in this case).
+
+```js
+import { render } from 'frint-react';
+
+window.app = new RouterApp();
+
+render(
+  window.app,
+  document.getElementById('root')
+);
+```
+
+Now if you open it in a browser you should see a page with \[Home] and \[About] links and 
+'Home page' content. If you click the links content will change accordingly.
+
+## Default route
+
+Another useful feature of `frint-router-react` is default route which allows you to render a 
+component or an app when no other route matched. To enable this behaviour add a `Route` without a 
+`path` to your group of `Route` components and them wrap them into `Switch`.
+
+```js
+...
+import { Route, Link, Switch } from 'frint-router-react';
+...
+const NotFoundPage = () => <div>Not found</div>;
+...
+const RootComponent = () => {
+  return (
+    <div>
+      ...
+      <div className="content">
+        <Switch>
+          <Route path="/" component={HomePage} exact />
+          <Route path="/about" component={AboutPage} />
+          <Route component={NotFoundPage} />
+        </Switch>
+      </div>
+    </div>
+  );
+};
+
+```  
+
+Now whenever you navigate to URL that doesn't match `/` or `/about` you'll see `NotFoundPage`
+rendered.
+
+Note: in `Switch` only the first matching `Route` gets rendered.
+
+## Nested routes
+
+It is also possible for child components to define their own subroutes. So for example if you want
+`AboutPage` to have subpages you can render `Link` and `Route` to it.
+
+```js
+...
+const AboutUs = () => <article>Some content about us...</article>;
+const AboutThem = () => <article>Some content about them...</article>;
+
+const AboutPage = ({ match }) => {
+  return (
+    <div>
+      <h2>About page</h2>
+        {<ul>
+          <li><Link to={`${match.url}`}>About</Link></li>
+          <li><Link to={`${match.url}/us`}>About us</Link></li>
+          <li><Link to={`${match.url}/them`}>About them</Link></li>
+        </ul>}
+        <Switch>
+          <Route path={`${match.url}/us`} component={AboutUs} />
+          <Route path={`${match.url}/them`} component={AboutThem} />
+        </Switch>
+      </div>
+    );
+  };
+...
+```
+
+Because `AboutPage` is rendered using `Route` component it receives a `match` object as a prop. This
+object contains `url` which let's you define routes relative to the current.  
+
+It is important that the parent `Route` doesn't have `exact` prop so that it would match all the URLs
+starting with the `path`.
+
+## Another example
+
+For more information about usage of Frint routing you can take a look at example here: 
```

Comment:
I haven't dealt with cli yet. So I've added step-by-step guide with list of commands. Maybe tomorrow you could talk a bit about it.